daphne/src/messages.rs

```diff
@@ -4,7 +4,7 @@
 //! Messages in the DAP protocol.
```

Comment:
Please double check that all of the following changes match the spec:
- Move time out of Nonce struct
- New ReportMetadata struct and corresponding changes to Report and ReportShare
- New field in AggregateInitialzeReq (we call it the BatchParameter here)
- All length prefixes are bumped properly (16 -> 32)
- New Query and BatchSelector structs (these are identical in fact).
- Changes to AggregateShareReq, CollectReq, and CollectResp.